lib/Controllers/create.js

```diff
@@ -30,6 +57,9 @@ Create.prototype.write = function(req, res, context) {
         res.header('Location', location);
       }
 
+      // Reload for that all field are present
+      return instance.reload();
```

Comment:
Else, maybe we can reload only for when the model has an include ?
You prefer ?